packages/itinerary-body/src/util.ts

```diff
@@ -128,3 +129,22 @@ export function getPlaceName(
   // Default to place name
   return place.name;
 }
+
+/**
+ * Helper function that assembles values for flex pickup/dropoff messages.
+ */
+export function getFlexMessageValues(
+  info: FlexBookingInfo
+): {
+  hasLeadTime: boolean;
+  hasPhone: boolean;
+  leadDays?: number;
+  phoneNumber?: string;
+} {
+  return {
+    hasLeadTime: coreUtils.itinerary.isAdvanceBookingRequired(info),
```

Comment:
Potential improvement (to reduce the number of params in the language strings): could we remove `hasLeadTime` and use whether `leadDays` is non-zero instead? Same for `hasPhone` and `phoneNumber`.

Reply:
`hasLeadTime` is removed. `hasPhone` is not, as react-intl can to my knowledge not check the "blank-ness" of a string